include/fmt/core.h

```diff
@@ -3043,7 +3048,8 @@ template <typename Char, typename... Args> class basic_format_string {
              std::is_reference<Args>::value)...>() == 0,
         "passing views as lvalues is disallowed");
 #ifdef FMT_HAS_CONSTEVAL
-    if constexpr (detail::count_named_args<Args...>() == 0) {
+    if constexpr (detail::count_named_args<Args...>() ==
+                  detail::count_statically_named_args<Args...>()) {
```

Comment:
Doesn't this effectively make `fmt::arg` unusable?

Reply:
As far as I understand, `fmt::arg` is handled the same way as before. Consider the following table with results of boolean expressions inside this `if`:

|                   |`master`|PR     |
|-------------------|--------|-------|
|w/o named arguments| true   | true  |
|`fmt::arg`       | false  | false |
|`_a`               | false  | true  |
|`fmt::arg` + `_a`| false  | false |

This way, nothing should change for `fmt::arg`. In contrast with format string with a mix of named and non-named arguments.

Reply:
Makes sense.